### PR TITLE
Bookstore: initialize App with components 

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- Added React Router and set two s and s for the app's navigation
- Initialised React app.
- fix linters
- Update readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,183 @@
...
+1. `.github/workflows/linters.yml` is already included in project, this is used by Github to run tests on the repo.
+2. To install Webpack packages and dependencies
+
+- `npm init -y` initalize the project.
+- `npm i` installs all dependencies listed in the projects's package.json
+
+- `npm install webpack webpack-cli --save-dev` installs webpack
+- `npm install --save-dev html-webpack-plugin` installs webpack plugin
+- `npm install --save-dev style-loader css-loader` installs assets loaders
+- `npm install --save-dev webpack-dev-server` installs webpack server
```
Comment (Ismailco) [inline README.md]: - [ ] There is no need for the user to install Webpack, kindly consider removing this section from the README so it doesn't create confusion

File: README.md [hunk 2]
```diff
@@ -1,70 +1,183 @@
...
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcomed!
+
+Feel free to check the [issues page]().
```
Comment (Ismailco) [inline README.md]: - [ ] This issue link is broken kindly make sure to include the correct link here so it redirect to the issue page

File: src/components/BookItem.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const { title, author } = book;
+  return (
+    <div>
+      <h2>
+        {title}
+      </h2>
+      <h2>
+        {author}
+      </h2>
+    </div>
```
Comment (Ismailco) [inline src/components/BookItem.js]: - [ ] This component should contain a button to remove the correspondent book from the list of books

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Moses-chibuike, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Used CRA to create the app
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem.js (hunk 3 above)
Comment (Moses-chibuike) [reply src/components/BookItem.js]: Thanks for your review, check the ui provided and re-iterate please, there is no remove button required.

Comment (Moses-chibuike) [reply src/components/BookItem.js]: The button that should be visible at this stage is only the ADD BUTTON, which I have showcased

Comment (Ismailco) [reply src/components/BookItem.js]: @Moses-chibuike, it's clear in the requirement that the book component should have a remove button 👇🏼 

Screenshot_20230331_031120

**Review by pinkmoon25 — APPROVED** [review]
Hi @Moses-chibuike , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Gitflow :heavy_check_mark: 
Changes implemented :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**